integration/aggregate_documents_compat_test.go

```diff
@@ -228,8 +228,7 @@ func testAggregateCommandCompat(t *testing.T, testCases map[string]aggregateComm
 					return
 				}
 				require.NoError(t, compatErr, "compat error; target returned no error")
-
-				AssertEqualDocumentsSlice(t, compatRes, targetRes)
+				require.Equal(t, compatRes, targetRes)
```

Comment:
So, the result is a document, not a slice, right? Should we use `AssertEqualDocuments`?

Reply:
Yes they are documents. Don't we used `AssertEqualDocuments` when response contains special bson that cannot be compared with `Equal`? I thought testify package assertions were preferred when it works :thinking:

Reply:
I'm not sure anymore 😅 
@AlekSi what is preferred to check responses - to use `assert.Equal` as much as possible or to use `AssertEqualDocuments` as much as possible and why?

Reply:
I really don't have strong opinion on this, let's just use `AssertEqualDocuments` as suggested. See the update :hugs: